src/implementations.jl

```diff
@@ -1,5 +1,9 @@
 const use_stdcall = startswith(basename(libmpi), "msmpi")
 
+if Sys.isunix()
+    Libdl.dlopen(libmpi, Libdl.RTLD_LAZY | Libdl.RTLD_GLOBAL)
```

Comment:
Just wondering - In #450 I put the `dlopen` call in the `MPI.__init__` function to ensure it only gets called once. Is this a worry? Perhaps not?

Reply:
Yeah this change should not be necessary since:

https://github.com/JuliaParallel/MPI.jl/blob/1ebbbd9cf987ecd6061b4cb19cde426d14733e6f/src/MPI.jl#L64

Reply:
So you are loading the library twice ... here and in `MPI.__init__`. Actually, I don't think that hurts anything.

Reply:
It is here because this file is also included from `deps/build.jl`, and we evaluate `Get_library_version()` at precompile time (before `__init__` is called).